internal/plugins/scorecard/init.go

```diff
@@ -270,3 +286,36 @@ func makeDefaultOLMTestConfigs(testImageTag string) (cfgs []v1alpha3.TestConfigu
 
 	return cfgs
 }
+
+// newCustomConfigurationPatch returns default "custom" test configurations as JSON patch objects
+// to be inserted into the componentconfig base as a first stage test element.
+// The returned patches are intended to be marshaled and written to disk as in a kustomize patch file.
+func newCustomConfigurationPatch(testImageTag string) (ps jsonPatches) {
+	for _, cfg := range makeDefaultCustomTestConfigs(testImageTag) {
+		ps = append(ps, jsonPatchObject{
+			Op:    "add",
+			Path:  defaultJSONPath,
+			Value: cfg,
+		})
+	}
+	return ps
+}
+
+// makeDefaultCustomTestConfigs returns all default "custom" test configurations.
+func makeDefaultCustomTestConfigs(testImageTag string) (cfgs []v1alpha3.TestConfiguration) {
+	for _, testName := range []string{
+		"customtest1",
+		"customtest2"} {
```

Comment:
I assume that it is just to test and shows to the users how it works. 
However, it is in the init.go which make not it clear.